README.md

```diff
@@ -48,15 +50,30 @@ Options:
   -?, -h, --help  Show help and usage information
 
 Commands:
-  generate-script  Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally 
-                   modify the script if desired before running it.
-                   Note: Expects GH_SOURCE_PAT or GH_PAT env variable to be set.
-  migrate-repo     Invokes the GitHub API's to migrate the repo and all PR data.
-                   Note: Expects GH_PAT and GH_SOURCE_PAT env variables to be set. GH_SOURCE_PAT is optional, if not set GH_PAT will be used 
-                   instead.
+  generate-script       Generates a migration script. This provides you the ability to review the steps that this tool will take, and optionally modify the script if desired before running it.
+  grant-migrator-role   Allows an organization admin to grant a USER or TEAM the migrator role for a single GitHub organization. The migrator role allows the role assignee to perform migrations into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
+  migrate-repo          Invokes the GitHub APIs to migrate the repo and all repo data.
+  revoke-migrator-role  Allows an organization admin to revoke the migrator role for a USER or TEAM for a single GitHub organization. This will remove their ability to run a migration into the target organization.
+                        Note: Expects GH_PAT env variable to be set.
 ```
 
-To generate a script, you'll need to set an `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
+To generate a migration script, you'll need to set `GH_PAT` as an environment variable for your destination and `GH_SOURCE_PAT` for your source location. 
+
+## GitHub Enterprise Server (GHES) to GitHub Migration Usage
+
+GEI-CLI is a cross-platform .NET Core console application. General usage will use the `generate-script` command to create a script that can be used to migrate all repositories from a GHES organization.
```

Comment:
Looks like this line is a dupe from above

Reply:
That line is also at the start of the ADO section. I assumed it was there in case someone looking to do an ADO migration isn't reading the gh->gh section, so I added it for GHES too

Reply:
Hmm, I think the first sentence is unnecessary at repeating, but the second sentence could make sense to have. Does that sound reasonable to you ?

Reply:
Threw it up to the top, lemme know what you think https://github.com/github/gh-gei/pull/249/commits/632c1326cad86d6d8a77e1abba54d9e5ad95cc2c